<?xml version='1.0'?>
<launch>
    <arg name="port_name" default="can0"/>
    <arg name="simulated_robot" default="false"/>
    <arg name="odom_topic_name" default="odom"/>
    <arg name="pub_odom_tf" default="false" />
    <arg name="robot_type" default="ranger" />


    <node name="ranger_base_node" pkg="ranger_base" type="ranger_base_node" output="screen">
        <param name="port_name" type="string" value="$(arg port_name)" />
        <param name="simulated_robot" type="bool" value="$(arg simulated_robot)" />
        <param name="odom_frame" type="string" value="odom" />
        <param name="base_frame" type="string" value="base_link" />
        <param name="odom_topic_name" type="string" value="$(arg odom_topic_name)" />  
        <param name="pub_odom_tf" type="bool" value="$(arg pub_odom_tf)" />
        <param name="robot_type" type="string" value="$(arg robot_type)" />
    </node>
</launch>
